sdk/inc/azure/iot/az_iot_hub_client.h

```diff
@@ -47,6 +35,17 @@ enum
   AZ_HUB_CLIENT_DEFAULT_MQTT_TELEMETRY_QOS = 0
 };
 
+/**
+ * @brief Azure IoT Hub Client option values.
+ *
+ */
+typedef enum
+{
+  AZ_IOT_HUB_CLIENT_OPTION_VALUE_UNDEFINED = 0,
+  AZ_IOT_HUB_CLIENT_OPTION_VALUE_TWIN_CONTENT_TYPE_CBOR = 1,
+  AZ_IOT_HUB_CLIENT_OPTION_VALUE_TWIN_CONTENT_TYPE_JSON = 2,
+} az_iot_hub_client_option_values;
```

Comment:
I think this should be named something more specific to this feature. Something like `az_iot_hub_client_content_type` or `az_iot_hub_client_content_type_value`. At the very least here the `s` should be dropped from the end.

Reply:
i like it